signatures/mappings.py

```diff
@@ -0,0 +1,280 @@
+mapping = {
```

Comment:
This `mapping` dictionary is not used in the main codebase. If I do not import it, nothing breaks and SAGE produces the same AGs as before. Should I keep this or can it safely be removed?